api_specs/v2/component_analyses.yaml

```
@@ -106,8 +134,166 @@ paths:
+                  cvss: "1.3"
+                  is_private: false
+            recommended_version : "2"
+            message: "3 vulnerabilities within this dependency, 0 exploitable vulnerabilities"
+            highest_severity: "high"
+            public_vulnerabilities_count: 2
+            private_vulnerabilities_count: 1
+            exploitable_vulnerabilities_count: 0
+            vendor_package_link: "https://snyk.io/vuln/maven:org.zeroturnaround%3Azt-zip"
+        - package: "mango"
+          version: "2.1"
+          recommendation:
+            component_analyses:
+              vulnerability:
+                - cve_id: "CVE-2018-1002"
+                  cvss: "1.2"
+                  is_private: true
+                - cve_id: "CVE-2010-3082"
+                  cvss: "1.3"
+                  is_private: false
+            recommended_version : "2"
+            message: "3 vulnerabilities within this dependency, 0 exploitable vulnerabilities"
+            highest_severity: "high"
+            public_vulnerabilities_count: 2
+            private_vulnerabilities_count: 1
+            exploitable_vulnerabilities_count: 0
+            vendor_package_link: "https://snyk.io/vuln/maven:org.zeroturnaround%3Azt-zip"
+    no-vul-batch:
+      value:
+        - recommendation: {}
+
+    registered-user:
+      value:
+        package: "django"
+        version: "1.1"
+        recommendation:
+          component_analyses:
+            vulnerability:
+              - cve_id: "CVE-2018-1002"
+                cvss: "1.2"
+                is_private: true
+              - cve_id: "CVE-2010-3082"
+                cvss: "1.3"
+                is_private: false
+          recommended_version : "2"
+          message: "3 vulnerabilities within this dependency, 0 exploitable vulnerabilities"
+          highest_severity: "high"
+          public_vulnerabilities_count: 2
+          private_vulnerabilities_count: 1
+          exploitable_vulnerabilities_count: 0
+          vendor_package_link: "https://snyk.io/vuln/maven:org.zeroturnaround%3Azt-zip"
+
+    free-user:
+      value:
+        package: "mango"
+        version: "2.1"
+        recommendation:
+          component_analyses:
+            vulnerability:
+              - cve_id: "CVE-2018-1002"
+                cvss: "1.2"
+                is_private: true
+              - cve_id: "CVE-2010-3082"
+                cvss: "1.3"
+                is_private: false
+          recommended_version : "2"
+          message: "2 vulnerabilities within this dependency"
+          highest_severity: "high"
+          public_vulnerabilities_count: 2
+          private_vulnerabilities_count: 1
+          registeration_link: "https://app.snyk.io/login"
+
+    no-vul:
+      value:
+        recommendation: {}
+
   schemas:
+    RequestBody:
+      title: CA Batch Call Request Body
+      type: object
+      properties:
+        ecosystem:
+          type: string
+          example: pypi
```

This must be a enum,

```suggestion
        ecosystem:
          type: string
          enum:
            - maven
            - pypi
            - npm
```

---

Ack.